pkg/apis/metrics/v1alpha1/appmetricsconfig_types.go

```diff
@@ -12,6 +12,9 @@ type AppMetricsConfigSpec struct {
 // AppMetricsConfigStatus defines the observed state of AppMetricsConfig
 // +k8s:openapi-gen=true
 type AppMetricsConfigStatus struct {
+
+	//app host url
```

Comment:
The convention in Go for doc comments, is to start it with the name of the thing that's being documented. so in this case I'd suggest:

```go
// Host is the exposed URL for the AppMetricsService instance
```

Reply:
Fixed, will remember.